# Lane
Header header
std_msgs/Int32 id
std_msgs/Int32 left_id
std_msgs/Int32 right_id
std_msgs/Int32 next_id  # 如果有的话
geometry_msgs/Point[] points  #离散的samples，车辆行进方向